parser/parser.y

```diff
@@ -699,6 +699,9 @@ import (
 	SelectStmtFieldList		"SELECT statement field list"
 	SelectStmtLimit			"SELECT statement optional LIMIT clause"
 	SelectStmtOpts			"Select statement options"
+	SelectStmtPart1			"SELECT statement part 1"
```

Comment:
What's the difference between part1/2/3?

Reply:
They are syntax segments that abstracted from `SelectStmt` that can be reused in `UnionStmt`, each of them corresponds to a syntax branch of `SelectStmt`.

Reply:
How about changing to a better name?
`SelectStmtPart1`  `SelectStmtPart2`  `SelectStmtPart3` is not informative, maybe we can use
`SelectStmtBasic` `SelectStmtFromDual` `SelectStmtFromTable`

Reply:
@tiancaiamao agree